jmxtrans-core/src/test/java/com/googlecode/jmxtrans/jmx/JmxResultProcessorTest.java

```diff
@@ -237,13 +238,17 @@ public void canReadCompositeData() throws MalformedObjectNameException, Attribut
 		AttributeList attr = ManagementFactory.getPlatformMBeanServer().getAttributes(
 				memory.getObjectName(), new String[]{"HeapMemoryUsage"});
 		List<Result> results = new JmxResultProcessor(
-				dummyQueryWithResultAlias(),
+				Query.builder()
```

Comment:
Can you add a test rathen modifying an existing one? Can we have both tests with and without keys?

Reply:
Added separate test